files/2da/2da_bgee/summlimt.htm

```diff
@@ -17,3 +17,26 @@
   </div>
 </div>
 
+{% capture note %}
+<ul>
+  <li>
+    When engine is summoning any creature, checks for existing creatures that match:<br />
+    <pre>(EA <= 15 and EA != 3) and (GENDER == 6 or GENDER == 9)</pre><br />
```

Comment:
These EA values would be better as their named counterparts. The way it's written now it also gives the impression that the limit is global, eg. if the enemy summoned something, it would affect the player's count. But AFAIK, this limit has been there only for the party.

Reply:
> These EA values would be better as their named counterparts.

Done (but keep in mind that there's no symbolic name associated with `15`...).
I also specified that these limits hold only for the Party.

Reply:
We have it as "controllable", but too bad it's not in the ids.

Reply:
If you meant `CONTROLLED`, then it's `5` on EEs...

Reply:
No, it's a limit value like GOODCUTOFF, since all the normally controlable stuff is lower (pcs, charmed, dominated, familiars).

Reply:
I see.

Anyway, as you said, since it's not listed in `EA.IDS`, I'll leave it as `15`...

Reply:
Up to you, doesn't matter that much. Weidu can't use the symbol without it being in an IDS and until people start adding it, there's no damage in skipping it.